### PR TITLE
RHBPMS-5019: Job list: Job request details broken when command throws…

### PR DESCRIPTION
**Pull request by nmirasch**
… exception

Removed fileset that was causing the overflow problem in firefox and replace the text including it in a textarea.

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai @bsremac  could you take a look? Thanks!

File: jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopup.java [hunk 1]
```diff
@@ -128,6 +127,9 @@ public String getValue(RequestParameterSummary rowObject) {
                                                                                        paramValueColumn));
 
         this.dataProvider.addDataDisplay(executionParametersGrid);
+
+        errorsOccurredList.setVisibleLines(10);
```
Comment (bsremac) [inline jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopup.java]: Would only 10 lines of stack trace be enough to troubleshoot the error?
I guess it's not possible to make the number of lines shown dynamic i.e. to expand to fit the whole stack trace, right?

File: jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopup.java [hunk 2]
```diff
@@ -164,9 +166,11 @@ public void setRequest(RequestSummary r,
             errorControlGroup.setVisible(true);
             String html = "";
             for (ErrorSummary error : errors) {
-                html += "<strong>" + error.getMessage() + "</strong><br/>" + error.getStacktrace() + "<br><br>";
+                html += error.getMessage() +
```
Comment (bsremac) [inline jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopup.java]: Maybe it makes sense for easier readability reasons to also rename the variable from ```html`` to, for example, ```textArea``` so it's aligned with the field type used.

File: jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopupTest.java [hunk 3]
```diff
@@ -58,6 +57,8 @@ public void errorGenerationTest() {
                                    errors,
                                    null);
 
-        verify(errorsOccurredList).setHTML(SafeHtmlUtils.fromTrustedString("<strong>" + error.getMessage() + "</strong><br/>" + error.getStacktrace() + "<br><br>"));
+        verify(errorsOccurredList).setText("errorMessage\n" +
+                                                   "stackTrace\n" +
+                                                   "\n");
```
Comment (bsremac) [inline jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopupTest.java]: Just a cosmetic thing here. This String breakdown looks a bit off. Maybe fixing the indention so each part is exactly below the previous one would make it look nicer :)

**Comment by nmirasch** [comment]
@bsremac the problem was with the fileset at firefox. I have removed it and fixes the Execution Parameters table overflow. I think this details popup have to be changed in future to be consistent with the other details views.
The 10 lines limit is to be according to the text area that shows the stacktrace at Execution error view and to avoid the big scrolling at the popup.
I'm going to rename the field and do the cosmetic thing . I'll let you know when its done. Thanks!

File: jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopup.java (hunk 1 above)
Comment (jhrcek) [reply jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopup.java]: @bsremac I think Neus implemented this correctly. the "visible lines" only refers to the number of rows the textarea will display. It doesn't "trim the stacktrace". The remaining lines are still accessible via textarea's scrollbar as demonstrated [here](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_textarea). I think that it's ok as it is - just show 10 lines and let user resize the area (= default browser behaviour).

**Comment by nmirasch** [comment]
@bsremac changes done, could you take another look?

**Pull request merged**